src/tests/test_syncify.py

```diff
@@ -184,36 +186,34 @@ def wrapper():
     assert x == [7, 7]
 
 
+@requires_jspi
 @pytest.mark.requires_dynamic_linking
 @pytest.mark.xfail(reason="Will fix in a followup")
-def test_syncify_ctypes():
-    selenium.run_js(  # type: ignore[name-defined] # noqa: F821
-        """
-        await pyodide.runPythonSyncifying(`
-            from pyodide.code import run_js
-
-            test = run_js(
-                '''
-                (async function test() {
-                    await sleep(1000);
-                    return 7;
-                })
-                '''
-            )
+@run_in_pyodide
+def test_syncify_ctypes(selenium):
```

Comment:
It looks like this change is making the playwright hang. I'm not sure why exactly, but it's worth investigating (maybe it's stuck on getting the result of the run_in_pyodide?)

Would you mind skipping the test (instead of xfail) until we find the reason?

Reply:
> Would you mind skipping the test (instead of xfail) 

Not at all. For what it's worth, I was also having issues with it hanging when running locally. I wasn't really aware that xfail and skip were different to be honest.